framework/helpers/BaseHtml.php

```diff
@@ -660,12 +660,20 @@ public static function fileInput($name, $value = null, $options = [])
      * the attributes of the resulting tag. The values will be HTML-encoded using [[encode()]].
      * If a value is null, the corresponding attribute will not be rendered.
      * See [[renderTagAttributes()]] for details on how attributes are being rendered.
+     * The following special options are recognized:
+     *
+     * - `doubleEncode`: whether to encode HTML entities in `$value`. If `false`, HTML entities in `$value` will not
```

Comment:
you don't have to answer that to me ;) It's what developers are questioning if they see the parameter 'doubleEncode'